### PR TITLE
feat: Composer module improvement

### PR DESCRIPTION
**Pull request by diegolnasc**
Currently, the environment has few customization options. Therefore, I added some features that allow flexibility and deploy in an environment with shared vpc.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_whitespace
 Failed whitespace check. More details below. 
 ```bash 
Checking for trailing whitespace
Checking for missing newline at end of file
Error: No newline at end of file ./modules/create_environment/main.tf
Error: No newline at end of file ./modules/create_environment/variables.tf
 ``` 
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/simple_composer_env/main.tf
--- old/examples/simple_composer_env/main.tf
+++ new/examples/simple_composer_env/main.tf
@@ -31,6 +31,6 @@
   project_id        = var.project_id
   composer_env_name = var.composer_env_name
   region            = var.region
-  network = var.network
-  subnetwork = var.subnetwork
+  network           = var.network
+  subnetwork        = var.subnetwork
 }
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by diegolnasc** [comment]
Any updates?

File: examples/simple_composer_env/main.tf [hunk 1]
```diff
@@ -14,23 +14,16 @@
  * limitations under the License.
  */
 
-/******************************************
-  Provider configuration
- *****************************************/
-provider "google" {
```
Comment (bharathkkb) [inline examples/simple_composer_env/main.tf:20]: We should keep the provider version constraint in example

**Review by bharathkkb — COMMENTED** [review]
@kunalkg11 PTAL

File: examples/simple_composer_env/main.tf (hunk 1 above)
Comment (kunalkg11) [reply examples/simple_composer_env/main.tf:20]: Yes, I agree. We should keep the provider version in the example.

Comment (diegolnasc) [reply examples/simple_composer_env/main.tf]: Done.

**Comment by kunalkg11** [comment]
LGTM

**Review by kunalkg11 — APPROVED** [review]

**Pull request merged**